manifests/03-role.yaml

```diff
@@ -31,3 +32,29 @@ rules:
   - statefulsets
   verbs:
   - "*"
+- apiGroups:
+  - route.openshift.io
+  resources:
+  - routes
+  - routes/custom-host
+  verbs:
+  - "*"
+- apiGroups:
+  - batch
+  resources:
+  - cronjobs
+  verbs:
+  - "*"
+
+---
+kind: ClusterRole
```

Comment:
@jcantrill @smarterclayton for cluster role and role bindings should we be specifying that in our manifest or communicating that with the privileged operator?

Reply:
I defer to @smarterclayton and @derekwaynecarr  but i presume if the role is privileged we need to defer to the privileged operator

Reply:
In your manifest if you think only static changes are necessary.  It’s easy to move from manifest to operator if necessary.

Reply:
Okay that makes sense. 
@jcantrill the role isn't privileged but it is a cluster role definition to create priority classes. So long as OLM (or whomever it is that does operations with our manifest files) can create a cluster role and cluster role binding then we should be all set here.